scripts/sysman/fabric.yml

```diff
@@ -185,26 +185,6 @@ members:
       name: "txCounter"
       desc: "[out] Monotonic counter for the number of bytes transmitted (sum of all lanes). This includes all protocol overhead, not only the GPU traffic."
 --- #--------------------------------------------------------------------------
-type: struct
-desc: "Fabric Port Error Counters"
-class: $sFabricPort
-name: $s_fabric_port_error_counters_t
-base: $s_base_properties_t
```

Comment:
do we need to remove this as well? My thinking was if issue is only with `ZES_MAX_RAS_ERROR_CATEGORY_COUNT`, we just need to remove the last 3 ras error categories?

Reply:
@aravindksg it is better to do a clean revert, instead of pieces of it. These were part of the same commit where the new RAS errors were introduced, so assumption is that they are somehow related. If they were not, then they should have gone originally on different commits.